torch/onnx/_internal/fx/function_dispatcher.py

```diff
@@ -230,5 +230,7 @@ def _create_onnx_friendly_decomposition_table() -> (
 
 # This is a subset of PyTorch's built-in aten-to-aten decomposition. If an aten
 # op (e.g., torch.ops.aten.add.Tensor) has exporter, we exclude the op's decomposition
-# function in the _ONNX_FRIENDLY_DECOMPOSITION_TABLE.
-_ONNX_FRIENDLY_DECOMPOSITION_TABLE = _create_onnx_friendly_decomposition_table()
+# function in the _DEFAULT_ONNX_EXPORTER_DECOMPOSITION_TABLE.
+_DEFAULT_ONNX_EXPORTER_DECOMPOSITION_TABLE: Dict[
```

Comment:
```suggestion
DEFAULT_ONNX_EXPORTER_DECOMPOSITION_TABLE: Dict[
```

I would make public if it is used in other modules